src/main/java/io/vinyldns/java/model/membership/Group.java

```diff
@@ -0,0 +1,84 @@
+package io.vinyldns.java.model.membership;
+
+import java.util.Set;
+import org.joda.time.DateTime;
+
+public class Group {
+    private String name, email, description, id;
```

Comment:
mark the non optional fields (fields that are set in all the constructors) as final